Frameworks/UIKit/UIScrollView.mm

```diff
@@ -602,16 +607,19 @@ - (void)setScrollEnabled:(BOOL)enable {
 }
 
 - (void)_setScrollEnabled:(BOOL)enable {
-    _scrollViewer.HorizontalScrollBarVisibility(enable ? _previousHorizontalScrollBarVisibility : Controls::ScrollBarVisibility::Disabled);
-    _scrollViewer.VerticalScrollBarVisibility(enable ? _previousVerticalScrollBarVisibility : Controls::ScrollBarVisibility::Disabled);
+    _scrollViewer.HorizontalScrollBarVisibility(enable ? _previousHorizontalScrollBarVisibility : Controls::ScrollBarVisibility::Hidden);
```

Comment:
Yes, iOS doesn't show the ScrollIndicators (ScrollBars) when scrollEnabled = NO.

---
In reply to: [112270292](https://github.com/Microsoft/WinObjC/pull/2565#discussion_r112270292) [](ancestors = 112270292)